src/esp/gfx/Renderer.cpp

```diff
@@ -79,7 +82,27 @@ struct Renderer::Impl {
                 esp::gfx::Renderer::Impl::RendererShaderType::
                     DepthTextureVisualizer);
 
+#ifdef CORRADE_TARGET_APPLE
```

Comment:
Please add a comment pointing as to why this is needed. IE pointing to this PR or containing a relevant explanation / bug report.

Reply:
Nit: you could define a preprocessor define like ENABLE_DEPTH_VISUALIZATION_WORKAROUND and use that everywhere in this file instead of CORRADE_TARGET_APPLE. It would make it easier to (1) test/investigate the old codepath on Mac, or (2) use the workaround on other platforms. You would define ENABLE_DEPTH_VISUALIZATION_WORKAROUND based on CORRADE_TARGET_APPLE.

Reply:
@eundersander this is a good point :+1: 

On the engine side I'm having named workarounds that can be disabled at runtime, but that's probably an overkill here. (Though when we figure out the root cause, I'm planning to add such Apple-specific workaround.)